spdm.rflx

```diff
@@ -299,7 +304,53 @@ package SPDM is
                   and (Mut_Auth_Cap = False or Encap_Cap = True);
       end message;
 
-   -- 10.11 Measurement block
+   -- 10.11 GET_MEASUREMENTS request and MEASUREMENTS response message
+
+   --  The value 0 queries the responder for the total number of measurements
+   --  available. The value 255 requests all measurements. The values 1 to 254
+   --  request the measurement at the index corresponding to that value.
+   type Measurement_Operation is range 0 .. 255 with
+      Size => 8;
```

Comment:
Could we specify this property more strictly? Maybe an enumeration with `Always_Valid` set (and enumeration values for 0 and 255)?

Reply:
Do we get the value of that field in SPARK when I set it as Enum with `Always_Valid` Because the value from 1 .. 254 is treated as a number that is relevant to the protocol.

Reply:
This should be possible, @treiher?

Reply:
The raw value of an always-valid enum can be accessed in SPARK. It is not (yet) possible to access the raw value in a message or session specification.

Reply:
I guess that will do for now, as you should be able to handle the specified enum values first and do the `Always_Valid` case in your final transition in a session. @jklmnn Please try the enum, we'll see what we need to do for the sessions later...

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/82/commits/5d999833502c2772d950b556c565ea75cf54d8d0.